lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -304,8 +304,13 @@ def _section_student_admin(course, access):
         'get_student_progress_url_url': reverse('get_student_progress_url', kwargs={'course_id': unicode(course_key)}),
         'enrollment_url': reverse('students_update_enrollment', kwargs={'course_id': unicode(course_key)}),
         'reset_student_attempts_url': reverse('reset_student_attempts', kwargs={'course_id': unicode(course_key)}),
+        'reset_student_attempts_for_entrance_exam_url': reverse('reset_student_attempts_for_entrance_exam',
```

Comment:
It would be nice if we could format these lines a little differently -- is it possible to drop the 'reverse' down to the line underneath to keep everything together?